src/parser.rs

```diff
@@ -20,8 +20,10 @@
 //! Parse encoded string into a struct for reassembly
 
 use itertools::Itertools;
-use std::collections::{BTreeMap, HashSet};
-use std::fmt;
+use std::{
+    collections::{BTreeMap, HashSet},
+    fmt,
+};
```

Comment:
More FMT magic?

Reply:
yep. this behaviour is configurable, but this the correct formatting according to your current `.rustfmt` file